source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs

```diff
@@ -2346,9 +2366,31 @@ public void SelectCover()
             }
         }
 
-        public void SelectBackground()
+        public void DropCover(DragEventArgs args) 
         {
-            var path = dialogs.SelectImagefile();
+            if (args.Data.GetDataPresent(DataFormats.FileDrop)) 

```

Comment:
Can you create a single unction for this? Seems like unnecessary code duplication for just one different line.